### ISSUE TITLE
Wrong url - Translation in Fallback Type: free

### ISSUE THREAD

**Issue by zkrechowiec** [georgringer/news#2526]
## Bug Report

**Current Behavior**
I have a site with multiple languages (EN as default, FR as secondary), when I translate a news item, the slug for my news item contains the path_segment EN ex:
EN: /newspress/detail-view/test-en
FR: /fr/actualites/vue-detaillee/test-en instead of /fr/actualites/vue-detaillee/test-fr

If I don't configure a fallback language (which is what I need on my site), I can't access the detailed view of my FR news (by going through the language menu from the EN detailed view or the FR list view) I get a 404.

If I add a fallback to the EN language still in free mode, then the detailed view works with the url /fr/actualites/vue-detaillee/test-en

My url rewriting is configured like this:
routeEnhancers:
  News:
    type: Extbase
    limitToPages:
      - 91
    extension: News
    plugin: Pi1
    routes:
      -
        routePath: '/{news-title}'
        _controller: 'News::detail'
        _arguments:
          news-title: news
    defaultController: 'News::list'
    defaults:
      page: '0'
    aspects:
      news-title:
        type: PersistedAliasMapper
        tableName: tx_news_domain_model_news
        routeFieldName: path_segment

And my language menu is generated by the LanguageMenuProcessor

Thanks in advance,
Zoïa

**Environment**
- TYPO3 version(s): 12.4.20
- news version: 11.4.2
- Is your TYPO3 installation set up with Composer (Composer Mode): yes

**Comment by ulrike-cosmoblonde** [comment]
Hi,
I can confirm this behaviour.
In freemode, the generated URLs to news detail pages are always using the original path segment and never the translated path segment. This leads to a 404 (unless a fallback language is configured). It would be better to get the correct language-aware URLs.

**Comment by georgringer** [comment]
I am not sure if this is really a news issue but a core issue?

**Comment by ulrike-cosmoblonde** [comment]
@georgringer I agree! It is a core behaviour concerning any data type and currently the only workaround I could find is to configure a fallback language even though the language mode is "free" and not "fallback"

**Comment by vitamys** [comment]
has this been reported as a core bug? are there any news on this?

**Comment by ursbraem** [comment]
We are experiencing this as well.

**Comment by cepheiVV** [comment]
- Can anybody test if this is also a problem with TYPO3 v13?
 - Are we sure this is a core issue and not some misconfiguration?
 - Do we have a forge issue for this? (I can create a new one if there is none yet...)

**Comment by cepheiVV** [comment]
In my List view I get this localized news item in French.
There's a German parent record in the default language

- German UID: 3596
- French UID:  3597

`_localizedUid` is shown as modified.

<img width="410" alt="Image" src="https://github.com/user-attachments/assets/3856fadd-f59d-4aef-bb31-7d651769296e" />

**Comment by runepiper** [comment]
@cepheiVV I can confirm your observation. We do have the exact same issue.

**Comment by cepheiVV** [comment]
I installed this diff with `cweagans/composer-patches` composer package.

in composer.json

```
...
	"extra": {
		"typo3/cms": {
			"web-dir": "public"
		},
		"patches": {
			"georgringer/news": {
				"Fix localized news slugs when TYPO3 is set to free mode": "patches/news-linkviewhelper.diff"
			}
		}
	},
...
```

[news-linkviewhelper.diff.txt](https://github.com/user-attachments/files/19787841/news-linkviewhelper.diff.txt)

**Comment by einpraegsam** [comment]
I'm currently facing the same issue in TYPO3 13 with fallbackType:free in additional languages. Overruling getUid() in model did the trick. 
This can be done like it's described in the official news documentation: https://docs.typo3.org/p/georgringer/news/main/en-us/Tutorials/ExtendNews/ProxyClassGenerator/Index.html
